recipe/meta.yaml

```diff
@@ -38,6 +38,7 @@ outputs:
         - setuptools
         - setuptools_scm >=4
         - setuptools_scm_git_archive
```

Comment:
`setuptools_scm_git_archive` is no more needed, see https://github.com/matplotlib/matplotlib/commit/a31a01623283f3672b79dc2736957b4fd49ebe07